benchmarks/browser/benchmark-append-signed.html

```diff
@@ -71,9 +71,9 @@ <h2>Results</h2>
             outputElm.innerHTML = `${queriesPerSecond} queries per second, ${totalQueries} queries in ${seconds} seconds - Log entries: ${log.length}<br>` + outputElm.innerHTML
             queriesPerSecond = 0
           }, 1000)
-
-          const identity = await Identities.createIdentity({ id: 'userA' })
-          log = new Log(ipfs, identity, { logId: 'browser-benchmark-append-signed' })
+          const identities = new Identities()
+          const identity = await identities.createIdentity({ id: 'userA' })
+          log = new Log(ipfs, identity, identities, { logId: 'browser-benchmark-append-signed' })
```

Comment:
Could we not keep the previous API in these kind of cases?

We had:
```js
const identity = await Identities.createIdentity({ id: 'userA' })
```

And now:
```js
const identities = new Identities()
const identity = await identities.createIdentity({ id: 'userA' })
```

...which could be done in a static method `createIdentities()`. Any reason not to?

Reply:
Ah, ignore the previous comment. I understand now as we pass the identities in the log.